lib/vanagon/platform.rb

```diff
@@ -221,11 +221,22 @@ def is_cisco_wrlinux?
       return !!@name.match(/^cisco-wrlinux-.*$/)
     end
 
-    # Utility matcher to determine is the platform is an osx variety
+    # Utility matcher to determine if the platform is an osx variety
     #
+    # @deprecated Please use is_macos? instead
     # @return [true, false] true if it is an osx variety, false otherwise
     def is_osx?
-      return !!@name.match(/^osx-.*$/)
+      warn "is_osx? is a deprecated method, please use #is_macos? instead."
+      is_macos?
+    end
+
+    # Utility matcher to determine if the platform is a macos or osx variety
+    # is_osx is a deprecated method that calls is_macos
+    # We still match for osx currently but this will change
+    #
+    # @return [true, false] true if it is a macos or osx variety, false otherwise
```

Comment:
Nitpick: This could totally say `@return [Boolean] true if it is a macos or osx variety, false otherwise`; yardoc supports `Boolean` as a shortcut around `true, false` here. I don't think whoever wrote a lot of these knew that at the time. We can fix it later.